### PR TITLE
community[minor]: Add Riza Python/JS code execution tool

### PR DESCRIPTION
**Pull request by andrewmbenton**
- **Description:** Add Riza Python/JS code execution tool
- **Issue:** N/A
- **Dependencies:** an optional dependency on the `rizaio` pypi package
- **Twitter handle:** [@rizaio](https://x.com/rizaio)

[Riza](https://riza.io) is a safe code execution environment for agent-generated Python and JavaScript that's easy to integrate into langchain apps. This PR adds two new tool classes to the community package.

### CODE REVIEW

**Review by eyurtsev — APPROVED** [review]

File: libs/community/langchain_community/tools/riza/command.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+)
+from langchain_core.pydantic_v1 import BaseModel, Field
+from langchain_core.tools import BaseTool, ToolException
+
+
+class ExecPythonInput(BaseModel):
+    code: str = Field(description="the Python code to execute")
+
+
+class ExecPython(BaseTool):
```
Comment (eyurtsev) [inline libs/community/langchain_community/tools/riza/command.py:21]: Consider adding inline documentation to the tool for users -- So it surfaces on the API reference. You can include in the doc string a code example as well

**Comment by andrewmbenton** [comment]
@eyurtsev looks like I may have disrupted this with my force push. Sorry. Anything I can do to get back on track?

**Review by eyurtsev — APPROVED** [review]

**Comment by eyurtsev** [comment]
Run `make format` in the root of the repo to lint the docs

**Comment by andrewmbenton** [comment]
> Run `make format` in the root of the repo to lint the docs

Took care of this @eyurtsev, should be good to go now.

**Comment by eyurtsev** [comment]
You can run make lint / make format / make test to catch these issues locally from within the community package (libs/community) as long as you set up the env (poetry install --with dev,lint,test)

File: libs/community/langchain_community/tools/riza/command.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    description: str = """Execute Python code to solve problems.
+
+    The Python runtime does not have filesystem access. You can use the httpx
+    or requests library to make HTTP requests. Always print output to stdout."""
+    args_schema: Type[BaseModel] = ExecPythonInput
+    handle_tool_error: bool = True
+
+    client: Any = None
+
+    def __init__(self, **kwargs) -> None:
```
Comment (eyurtsev) [inline libs/community/langchain_community/tools/riza/command.py:34]:
```suggestion
    def __init__(self, **kwargs: Any) -> None:
```

File: libs/community/langchain_community/tools/riza/command.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    The JavaScript runtime does not have filesystem access, but can use fetch
+    to make HTTP requests and does include the global JSON object. Always print
+    output to stdout."""
+    args_schema: Type[BaseModel] = ExecJavaScriptInput
+    handle_tool_error: bool = True
+
+    client: Any = None
+
+    def __init__(self, **kwargs) -> None:
```
Comment (eyurtsev) [inline libs/community/langchain_community/tools/riza/command.py:75]:
```suggestion
    def __init__(self, **kwargs: Any) -> None:
```

**Review by eyurtsev — COMMENTED** [review]
I suspect it's missing Any type annotation on the kwargs

**Comment by andrewmbenton** [comment]
> I suspect it's missing Any type annotation on the kwargs

Blerg sorry, just fixed.

> You can run make lint / make format / make test to catch these issues locally from within the community package (libs/community) as long as you set up the env (poetry install --with dev,lint,test)

For what it's worth, `make lint` within `libs/community` suggests changes to over 440 files, most of which are a single newline addition like below. Makes it hard to parse out things that I've modified.
```sh
--- tests/unit_tests/vectorstores/test_utils.py
+++ tests/unit_tests/vectorstores/test_utils.py
@@ -1,4 +1,5 @@
 """Test vector store utility functions."""
+
 import numpy as np
 from langchain_core.documents import Document
 

442 files would be reformatted, 1543 files already formatted
make: *** [Makefile:52: lint] Error 1
```

And `make test` fails (although admittedly I may have my environment set up incorrectly):
```sh
(venv) andrew@computer:~/riza/riza-io/langchain/libs/community$ make test
poetry run pytest --disable-socket --allow-unix-socket tests/unit_tests/
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --snapshot-warn-unused --disable-socket --allow-unix-socket
  inifile: /home/andrew/riza/riza-io/langchain/libs/community/pyproject.toml
  rootdir: /home/andrew/riza/riza-io/langchain/libs/community

make: *** [Makefile:19: test] Error 4
```

**Comment by eyurtsev** [comment]
> For what it's worth, make lint within libs/community suggests changes to over 440 files, most of which are a single newline addition like below. Makes it hard to parse out things that I've modified.

Make sure that you environment is up to date with the poetry lock file. If you see many requested changes, it's b/c it's using a different version of the same linter.
